lib/Maho.php

```diff
@@ -180,4 +180,25 @@ public static function errorReport(array $reportData = [], int $httpResponseCode
         header("HTTP/1.1 {$httpResponseCode} {$description}", true, $httpResponseCode);
         echo "<html><body><h1>There has been an error processing your request</h1>{$reportIdMessage}</body></html>";
     }
+
+    public static function getImageManager(array $customOptions = []): \Intervention\Image\ImageManager
+    {
+        $defaultOptions = [
+            'autoOrientation' => false,
+            'strip' => true,
+        ];
+        $options = [
+            ...$defaultOptions,
+            ...$customOptions,
+        ];
+
+        $driverClass = extension_loaded('ffi') === true && ini_get('ffi.enable') === '1' && ini_get('zend.max_allowed_stack_size') === '-1' && \Composer\InstalledVersions::isInstalled('intervention/image-driver-vips')
+            ? \Intervention\Image\Drivers\Vips\Driver::class // @phpstan-ignore class.notFound
+            : \Intervention\Image\Drivers\Gd\Driver::class;
+
+        return new \Intervention\Image\ImageManager(
+            $driverClass,
+            ...$options,
+        );
```

Comment:
```suggestion
        $driverClasses = [
            \Intervention\Image\Drivers\Gd\Driver::class,
        ];

        if (\Composer\InstalledVersions::isInstalled('intervention/image-driver-vips')) {
            // @phpstan-ignore class.notFound
            array_unshift($driverClasses, \Intervention\Image\Drivers\Vips\Driver::class);
        }

        foreach ($driverClasses as $driverClass) {
            try {
                return \Intervention\Image\ImageManager::withDriver($driverClass, ...$options);
            } catch (Intervention\Image\Exceptions\DriverException) {
            }
        }

        Mage::throwException('No image driver found');
```

This is much simpler :)